src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,126 @@
+
+<br/>
+<br/>
+
+*****
+
+## Placement
+On a user interface which is designed for a left-to-right script like English, action buttons of a footer (for example in a dialog) are typically located at the bottom-right corner.
+
+When there are two buttons placed together, the button that represents a positive action is always on the right side, and the one which presents a negative one is on the left.
+
+A positive action could be an action that:
+- takes the user forwards or to next step; e.g. *Next*, *Skip*, or *Continue*
+- is primary, and most probably the one that users is going to select, e.g. *Save*, *Update*, *Send* (or even *Delete* which is a negative notion, but can be an expected or "primary" action, depending on the context, like in a confirm deletion dialog)
+
+A negative action could be an action that:
+- takes the user back, dismisses a dialog, or cancels an ongoing process; e.g. *Back*, *Cancel*, *Discard*
+- is secondary, which means it is not likely that users clicks it in that context
+
+<limel-example-action-buttons-placement></limel-example-action-buttons-placement>
+
+:::tip
+In [`limel-dialog`](#/components/limel-dialog), there is a `slot` called `button`, which is flex-box. It automatically aligns its child elements to the right side. All you need to do is to write `slot="button"` on your elements.
+
+```tsx
+<limel-button
+    label="Ok"
+    slot="button"
+/>
+```
+:::
+
+<br/>
+
+*****
+
+## Primary versus secondary actions
+
+As you know, `limel-button` can have a `primary={true}` property, which makes it stand out by getting the primary color of your theme.
+
+One of the action buttons is normally the primary action. That is the action that you expect the users to take, or you want or prefer them to click on.
+
+<limel-example-action-buttons-primary-secondary></limel-example-action-buttons-primary-secondary>
+
+A primary button has a stronger visual weight, and appearing more distinct makes it easier for users to chose the correct, safe, or promoted action.
+
+You can use this visual distinction to promote the action that is safer, even if that action may not be the primary action. You can do so because sometimes:
+- it can be hard to identify the intention of the user
+- the action may be irreversible or risky
+
+<limel-example-action-buttons-primary-secondary-reversed></limel-example-action-buttons-primary-secondary-reversed>
+
+<br/>
+
+*****
+
+## Choosing good labels
+
+Labels you use should make sense together. You are usually dealing with contradictory actions. Therefore, what you put on each button should also reflect the contradiction.
+
+<limel-example-action-buttons-choosing-labels></limel-example-action-buttons-choosing-labels>
+
+Labels must be making each option as clear as possible. An explicit label give users more confidence in selecting the correct action.
+
+:::note
+- Make sure the label explains what the button does, rather than a generic label (such as “OK”, or "Yes").
```

Comment:
This sentence uses different grammatical forms before and after the comma.

```suggestion
- Use a label that explains what the button does, rather than a generic label (such as “OK”, or "Yes").
```